lib/theme_check/checks/liquid_tag.rb

```diff
@@ -35,14 +36,30 @@ def after_document(_node)
     def increment_consecutive_statements(node)
       @first_statement ||= node
       @consecutive_statements += 1
+      @consecutive_nodes << node
     end
 
     def reset_consecutive_statements
-      if @consecutive_statements >= @min_consecutive_statements
-        add_offense("Use {% liquid ... %} to write multiple tags", node: @first_statement)
+      if (@consecutive_statements >= @min_consecutive_statements) && @first_statement
+        add_offense("Use {% liquid ... %} to write multiple tags", node: @first_statement) do |corrector|
+          next if @first_statement.nil?
+          lines = @first_statement.source.split("\n").collect(&:rstrip)
+          # remove tags to be replaced by liquid tag
+          @first_statement.source.sub!("\n#{lines[@consecutive_nodes[1].line_number - 1, @consecutive_nodes[-1].line_number].join("\n")}", "")
```

Comment:
This method works but is there a better way of removing the tags that are now a part of the liquid tag?